includes/reader-revenue/woocommerce/my-account/class-my-account-ui-v0.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Connection with WooCommerce's features.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+use Newspack\WooCommerce_Connection;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Connection with WooCommerce's "My Account" page.
+ */
+class My_Account_UI_V0 extends WooCommerce_My_Account {
```

Comment:
Why does this class extend `WooCommerce_My_Account`? If `My_Account_UI_XX` classes are meant to inherit shared utility, I think it should be its own `My_Account_UI` base class.

Reply:
They're extending `WooCommerce_My_Account` so they can both use constants and methods from that class as `self::`, but I suppose they could just call `WooCommerce_My_Account::` directly and not extend any class. I think the `My_Account_UI_XX` classes are separate enough that they don't need to share any functions between them, so 2dd7f701 removes the extension and just makes these regular classes.

If they do eventually need to share some methods or constants, such as specific template parts, we can add the `My_Account_UI` base class.